lib/screens/stock_take_details/cash/cash_count.dart

```diff
@@ -16,53 +16,53 @@ class CashCount extends StatelessWidget {
               children: [
                 TableRow(children: [
                   getNoFieldText('200 Note'),
-                  getFieldText(),
+                  getFieldText(TextInputType.number),
```

Comment:
Why didn't you define the TextInputType in your getFieldText function but you are repeating it everywhere.